vscode_extension/src/helpers.ts

```diff
@@ -0,0 +1,23 @@
+import { workspace } from "vscode";
+
+export function ensure(test: () => boolean, failureMessage: string): void {
```

Comment:
Is there a runtime assert lib that would cover this? I looked briefly and it looks like it's really just testing libraries. We might be able to get away with using Node's assert library https://nodejs.org/api/assert.html if we didn't want to replicate assert calls.

Reply:
I'll look into this; thanks for the pointer!

Reply:
I've been looking through the assert library you pointed me to and, while I think it makes sense to use what's built-in, what I'm noticing when I use that is that instead of throwing an error that the user can see as a popup message (to indicate that there's an issue with their python version, for example) these assertion errors are showing up in the debug console, which a user wouldn't see unless they happen to be debugging the extension, which I'd assume most won't be.

As a result of this, I'm going to leave in the `ensure` stuff, for now, since I didn't find much in the way of alternatives to `assert`.